config/default.yml

```diff
@@ -525,6 +525,13 @@ Style/MethodName:
     - snake_case
     - camelCase
 
+Style/MultilineList:
```

Comment:
The cop is enabled by default but each part is disabled by default. I think it would be better to do it the other way around, because then people can find it when they look in `disabled.yml` for extra things to turn on.